src/test/resources/features/CreateEvent.feature

```diff
@@ -0,0 +1,67 @@
+@CreatingDifferentTypesOfEvents
+Feature: Create events
+  This feature deals with the create events functionality of the application
```

Comment:
Have you considered adding a BDD style description: 'As a ..., I want to..., So that...'